app/assets/javascripts/analytics.js.erb

```diff
@@ -136,9 +136,7 @@ var linkedDomains = [
 ]
 
 window.GOVUK.analyticsVars = window.GOVUK.analyticsVars || {}
-window.GOVUK.analyticsVars.gaProperty = '<%= Rails.application.config.ga_universal_id %>'
 window.GOVUK.analyticsVars.primaryLinkedDomains = ['account.gov.uk']
-window.GOVUK.analyticsVars.gaPropertyCrossDomain = '<%= Rails.application.config.ga_secondary_id %>'
 window.GOVUK.analyticsVars.linkedDomains = linkedDomains
```

Comment:
I wonder if we risk any problems if these variables aren't set for whatever reason and need to check that gaProperty is set or not before initialising analytics. If it fails gracefully all the better.

Reply:
Oh good question. I think we're good though:

https://sourcegraph.com/github.com/alphagov/govuk_publishing_components/-/blob/app/assets/javascripts/govuk_publishing_components/analytics/init.js?L8:16